### PR TITLE
INF-198 Moved login cypress commands to authentication submodule

### PR DESCRIPTION
**Pull request by Galzk**
Please merge on authentication repo before this one so the tests will pass
https://github.com/eScribers/authentication/pull/1

### CODE REVIEW

File: cypress/integration/twilio-video.spec.js [hunk 1]
```diff
@@ -25,7 +25,9 @@ const caseRef = uuid();
    before(() => { 
                   let userName = Cypress.env('loginAdminUserName');
                   let password = Cypress.env('loginAdminPassword');
-                  cy.login(conferenceUrlPath, userName, password);
+                  cy.visit(conferenceUrlPath);
```
Comment (talr) [inline cypress/integration/twilio-video.spec.js]: Move into the login function as loginUrlPath

File: cypress/integration/twilio-video.spec.js [hunk 2]
```diff
@@ -25,7 +25,9 @@ const caseRef = uuid();
...
+                  cy.login(userName, password);
+                  cy.url().should('include', conferenceUrlPath);
```
Comment (talr) [inline cypress/integration/twilio-video.spec.js]: Add after cy.login new command cy.visit(newConferenceUrlPath)

File: cypress/integration/twilio-video.spec.js [hunk 3]
```diff
@@ -25,7 +25,9 @@ const caseRef = uuid();
    before(() => { 
                   let userName = Cypress.env('loginAdminUserName');
```
Comment (talr) [inline cypress/integration/twilio-video.spec.js]: let managerUserName = Cypress.env('managerUserName')

File: cypress/integration/twilio-video.spec.js [hunk 4]
```diff
@@ -25,7 +25,9 @@ const caseRef = uuid();
...
                   let password = Cypress.env('loginAdminPassword');
```
Comment (talr) [inline cypress/integration/twilio-video.spec.js]: let managerPassword = Cypress.env('managerPassword')

**Review by talr — CHANGES_REQUESTED** [review]
Move all Authentication tests into authentication repository.
Implement test filenames naming convention.

**Comment by Jonniedev** [comment]
Looks good.
Does it run without errors?

**Comment by Galzk** [comment]
> 
> 
> Looks good.
> Does it run without errors?

Yup, did a live demo
I'll have to modify the PHP portion afterwards and I'll ping you when the PR is ready
Thanks

**Comment by Galzk** [comment]
Oh I miss-clicked

**Comment by Galzk** [comment]
@talr What's our procedure on changing/adding env vars?
We've got 2 modifications
```
managerUserName
managerPassword
```
It was 
```
loginAdminUserName
loginAdminPassword
```
But I don't know if that's safe to remove or is it used across different projects

**Review by talr — APPROVED** [review]

**Pull request merged**